modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1329 @@
+
+With this update, Enhanced Media Embed now wraps media elements in a responsive `<div>` container and moves the controlling `max-width` property to this outer container.
+
+This allows the embedded element to maintain its aspect ratio, as expected.
+
+For information on the **Enhanced Media Embed** plugin, see: xref:introduction-to-mediaembed.adoc[Introduction to Enhanced Media Embed].
+
+
+=== Mentions 3.0.4
+
+The {productname} 6.4.1 release includes an accompanyin release of the **Mentions** premium plugin.
+
+**Mentions** 3.0.4 includes the following fix:
+
+==== It wasn’t possible to create a new line immediately after a pasted Mention.
+//#TINY-9432
+
+Previously, if a _Mention_ was added to a {productname} editor; and the _Mention_ was selected; copied or cut; and then pasted on to a new line, attempting to then create a new line by pressing the **Return** key failed.
+
+As well, and depending on the host browser, the insertion point would not present in the editor until it was moved, by clicking elsewhere in the editor.
+
+This could be worked-around by moving the insertion point and then moving it back to immediately after the _Mention_ or by typing any character and then deleting that character.
+
+With this update, pressing **Return** immediately following the pasting of a _Mention_ string in a {productname} editor creates a new line, as expected.
+
+Also, the insertion point displays in all browsers immediately following a pasted _Mention_ string, also as expected.
+
+For information on the **Mentions** plugin, see xref:mentions.adoc[Mentions].
+
+
+=== Merge Tags 1.2.1
+
+The {productname} 6.4.1 release includes an accompanying release of the **Merge Tags** premium plugin.
+
+**Merge Tags** 1.2.1 includes the following change.
+
+==== The mceMergeTag command now preserves the selected content as a plain text and wraps it in a merge tag prefix and suffix if the content length exceeds the limit.
+//#TINY-9444
+
+In previous versions of {productname}, the `mceMergeTag` command would not wrap the selected content within the merge tag prefix and suffix when the content length exceeded the character limit for the merge tag.
+
+As a consequence, when the `text_patterns` feature was enabled and a user entered a too-long merge tag, the prefix and suffix of the tag were excluded from the inserted content.
+
+To address this problem, improvements to the `mceMergeTag` command were introduced with the release of {productname} 6.4.1. Now, if the length of the selected content exceeds the character limit for the merge tag, the content will be inserted as a plain text, including the prefix and suffix.
+
+[NOTE]
+If any content is embraced with a merge tag prefix and suffix but exceeds the character limit for merge tag content length, it will not be converted into an official merge tag.
+
+For information on the **Merge Tags** plugin, see: xref:mergetags.adoc[Merge Tags].
+
+
```

Comment:
there is another entry of `Merge Tag` in `line 169`

Reply:
```suggestion
```

Reply:
@ArvinJ-H removing duplicate, outdated version.
Good pickup.